doc/src/fix_external.rst

```diff
@@ -173,9 +173,17 @@ stress/atom <compute_stress_atom>` commands.  The former can be
 accessed by :doc:`thermodynamic output <thermo_style>`.  The default
 setting for this fix is :doc:`fix_modify virial yes <fix_modify>`.
 
-This fix computes a global scalar which can be accessed by various
-:doc:`output commands <Howto_output>`.  The scalar is the potential
-energy discussed above.  The scalar stored by this fix is "extensive".
+This fix computes a global scalar, a global vector, and a per-atom array
+which can be accessed by various :doc:`output commands <Howto_output>`.
+The scalar is the potential energy discussed above.  The scalar stored
+by this fix is "extensive".  The vector is a custom vector set by the
```

Comment:
I think you mean a global vector, not a custom vector?  The latter is a per-atom property.